tests/main/experimental-features/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: experimental features are exported by snapd
+details: |
+    Experimental features are exported as flag files that can be easily read by
+    snap-confine and snap-update-ns that otherwise don't have access to the
+    system state.
+execute: |
+    snap set core experimental.layouts=true
+    test -f /var/lib/snapd/features/layouts
+
+    snap set core experimental.layouts=false
+    test ! -f /var/lib/snapd/features/layouts
```

Comment:
(nitpick^2) most other tests seem to use ! at the start of the line for a negative test like this. Is there any particular advantage of one vs the other?

Reply:
This way `test` handles the negation but I only really do it because it reads more natural (to me) when written left-to-right. "test that it is not the case that there is a file named /var/lib/snapd/features/layouts". I don't have a strong preference, it's just how I wrote it.